openapi/paths/user_posts.yml

```
@@ -18,3 +18,31 @@ get:
             $ref: ../openapi.yml#/components/schemas/GetUserPostsTimelineResponse
     "500":
       description: Unexpected error occurred.
+
+post:
+  tags:
+    - X-Clone
+  summary: Creates a new post.
+  parameters:
+  - in: path
+    name: id
+    schema:
+      type: string
```

You can specify the format here.

```suggestion
      type: string
      format: uuid
```